tests/models/whisper/test_modeling_flax_whisper.py

```diff
@@ -0,0 +1,511 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import inspect
+import unittest
+
+from transformers import WhisperConfig, is_flax_available
+from transformers.testing_utils import require_flax, slow
+from transformers.utils import cached_property
+from transformers.utils.import_utils import is_datasets_available
+
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_flax_common import FlaxModelTesterMixin, floats_tensor
+
+
+if is_datasets_available():
+    import datasets
+    from datasets import load_dataset
+
+if is_flax_available():
+    import numpy as np
+
+    import jax
+    from transformers import (
+        FlaxWhisperForConditionalGeneration,
+        FlaxWhisperModel,
+        WhisperFeatureExtractor,
+        WhisperProcessor,
+    )
+
+
+@require_flax
+class FlaxWhisperModelTester:
+    config_cls = WhisperConfig
+    config_updates = {}
+    hidden_act = "gelu"
+
+    def __init__(
+        self,
+        parent,
+        batch_size=1,
+        seq_length=3000,
+        is_training=True,
+        use_labels=False,
+        vocab_size=99,
+        d_model=384,
```

Comment:
This is quite a large model for a test! Could we decrease it down to a similar size to that used in the PT test? https://github.com/huggingface/transformers/blob/2222740f50ef0443c69e0a5503f5ea9d6767be8e/tests/models/whisper/test_modeling_whisper.py#L84-L91